docs/dev/reputation-based-trade-amount.md

```
@@ -0,0 +1,106 @@
+The basic formula is:
+
+`Max allowed trade amount for a seller = Sellers reputation score / 200`
+
+Here is an overview of all use cases where this is applied to:
+
+## My offers
+
+At app startup we check if the user has sell offers which have a trade amount larger than their allowed reputation based amount. 
+If so, we show a popup to remove those offers.
+
+<img src="img/reputation-based-amounts/offers_with_too_high_amounts.png" width="1200"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)